### ISSUE TITLE
[openapi2kong] Server Port only supports up to 32767

### ISSUE THREAD

**Issue by danielpoonwj** [Kong/go-apiops#104]
Hi,

If we use a value larger than `32767` as the port in the url under `servers`, the service port gets set to `32767`. We should be able to specify ports up to `65535`.

**_openapi_**
```yaml
servers:
  - url: 'http://127.0.0.1:50000'
```

**_generated decK state_**
```
services:
  - port: 32767
...
```

It seems that it's because of this line in code:
https://github.com/Kong/go-apiops/blob/dab2c186e0db6e117c47d83b377c8d0cc9f8bde0/openapi2kong/service.go#L233

Setting bit size to 16 doesn't support any higher port numbers. There's actually an error that is returned but that is ignored.

**Comment by Tieske** [comment]
a PR would be most welcome :)

**Comment by Tieske** [comment]
Fixed in 0.1.24
